ui/src/reducers/tests/draftItem.test.js

```diff
@@ -0,0 +1,218 @@
+import * as actions from "../../actions/draftItem";
+import draftReducer from "../draftItem";
+import { Map, fromJS } from "immutable";
+
+// action draft
+const draft = {
+  access: null,
+  created: null,
+  created_by: null,
+  experiment: null,
+  files: Map({}),
+  id: null,
+  links: null,
+  metadata: {
+    additional_resources: {
+      internal_discussions: "blah blah"
+    },
+    basic_info: {
+      abstract: "This is the abstract",
+      conclusion: "This is the conclusion",
+      ana_notes: ["AN-2121/212"],
+      general_title: "This is the new title"
+    }
+  },
+  revision: null,
+  schema: null,
+  status: null,
+  type: null,
+  updated: null
+};
+
+// different states
+const initialState = Map({
+  errors: [],
+  schemaErrors: [],
+  actionsLayer: false,
+  actionsLayerType: null,
+  showPreviewer: false,
+  filePreviewEditLayer: true,
+  filePreviewEdit: {},
+  uploadFiles: Map({}),
+  pathSelected: null,
+
+  workflows: fromJS([]),
+  workflows_items: fromJS({}),
+  bucket: Map({}),
+  formData: null,
+  // From backend: deposit resource
+  access: null,
+  can_admin: false,
+  can_update: false,
+  created: null,
+  loading: false,
+  created_by: null,
+  experiment: null,
+  files: Map({}),
+  id: null,
+  links: null,
+  metadata: {},
+  revision: null,
+  schema: null,
+  schemas: null,
+  status: null,
+  type: null,
+  updated: null
+});
+const generalTitleStateBefore = Map({
+  errors: [],
+  schemaErrors: [],
+  actionsLayer: false,
+  actionsLayerType: null,
+  showPreviewer: false,
+  filePreviewEditLayer: true,
+  filePreviewEdit: {},
+  uploadFiles: Map({}),
+  pathSelected: null,
+
+  workflows: fromJS([]),
+  workflows_items: fromJS({}),
+  bucket: Map({}),
+  formData: {
+    additional_resources: {
+      internal_discussions: "blah blah"
+    },
+    basic_info: {
+      abstract: "This is the abstract",
+      conclusion: "This is the conclusion",
+      ana_notes: ["AN-2121/212"],
+      general_title: "This is the new title"
```

Comment:
`general_title` correct position is under `metadata` ( `metadata > general_title` )